data-tool/flows/tombstone/tombstone_utils.py

```diff
@@ -75,13 +75,13 @@ def format_address_data(address_data: dict, prefix: str) -> dict:
 
     formatted_address = {
         'address_type': address_type,
-        'street': street,
-        'street_additional': street_additional,
-        'city': address_data[f'{prefix}city'],
-        'region': address_data[f'{prefix}province'],
+        'street': street or '',
+        'street_additional': street_additional or '',
+        'city': address_data[f'{prefix}city'] or '',
+        'region': address_data[f'{prefix}province'] or '',
         'country': address_data[f'{prefix}country_typ_cd'],
-        'postal_code': address_data[f'{prefix}postal_cd'],
-        'delivery_instructions': delivery_instructions
+        'postal_code': address_data[f'{prefix}postal_cd'] or '',
+        'delivery_instructions': delivery_instructions or ''
```

Comment:
Updating according to https://github.com/bcgov/lear/pull/3060#discussion_r1830212132